packages/expo-cli/src/commands/publish/publishAsync.ts

```diff
@@ -150,6 +154,25 @@ function assertValidReleaseChannel(releaseChannel?: string): void {
   }
 }
 
+function assertExpoUpdatesInstalled(projectDir: string, sdkVersion?: string): void {
+  // before sdk 44, expo-update was included in with the expo module
+  if (sdkVersion && semver.lt(sdkVersion, '44.0.0')) {
```

Comment:
Probably want to use Versions from xdl since it accounts for `UNVERSIONED` and asserts when the semver is invalid in a controlled way.